includes/wf_crm_webform_postprocess.inc

```diff
@@ -627,6 +627,9 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     $params = $contact['contact'][1];
     // CiviCRM API is too picky about this, imho
     $params['contact_type'] = ucfirst($params['contact_type']);
+    if (!empty($contact['email'])) {
```

Comment:
@omarabuhussein I have added new `email` param in `createContact` method to avoid duplicate contact creation.
Here is the one scenario capture by Varun - When user try to create the new contact for existing contact field then 2 user gets create one with email address and one without email address which was not capture in `findDuplicateContact` method.
So I have added the email param to skip the duplicate contact in `findDuplicateContact` method

Reply:
Do we need to take into account Contact matching rules (dedupe rules) which can be configured per form? Or maybe that doesn't apply in this scenario?